lib/octokit/preview.rb

```diff
@@ -42,5 +42,15 @@ def warn_preview(type)
 header.
 EOS
     end
+
+    def self.get_media_type(preview)
+      PREVIEW_TYPES.each do |type|
+        if type.last.include? preview
+          return type.first
+        else
+          # octokit needs a new preview
```

Comment:
I'd love some suggestions for this and what you think of this approach in general 👀 it takes in the preview name used in the Open API spec and returns our Octokit abbreviation.